src/services/profile.service.js

```diff
@@ -0,0 +1,89 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../models/user.js';
+import { emailService } from './email.service.js';
+import { v4 as uuidv4 } from 'uuid';
+import { jwtService } from './jwt.service.js';
+import bcrypt from 'bcrypt';
+
+function getAllInfo() {
+  return User.findAll({
+    where: {
+      activationToken: null,
+    },
+  });
+}
+
+function normalize({ id, name, email, password }) {
+  return {
+    id,
+    name,
+    email,
+    password,
```

Comment:
Including the password in the normalize function can lead to security risks. Consider removing the password from the normalized user object.